apps/prairielearn/src/tests/sync/questionsSync.test.sql

```diff
@@ -0,0 +1,6 @@
+-- BLOCK set_sharing_name
+UPDATE pl_courses
+SET
+  sharing_name = $sharing_name
+WHERE
+  short_name = $course_name
```

Comment:
There's an existing `updateCourseSharingName` function. Consider using that instead of introducing a bespoke query.